variables.tf

```diff
@@ -195,3 +195,10 @@ variable "protocol_version" {
   default     = "HTTP1"
   type        = string
 }
+
+
+variable "efs_volumes" {
+  description = "Volumes definitions"
+  default     = []
+  type        = list(any)
```

Comment:
I think this could have an object specification, in order to make sure garbage doesn't get passed in.